examples/apps/cra-demo/src/App.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { KeyValueDataObject, KeyValueInstantiationFactory, IKeyValueDataObject } from "@fluid-experimental/data-objects";
+import { Fluid } from "@fluid-experimental/fluid-static";
+import { getContainerId } from "./getContainerId";
+
+// useKVPair is an example of a custom hook that returns Fluid backed state and a method to modify that state
+function useKVPair() {
```

Comment:
Is the developer expected to write a similar custom hook? Or are we exporting a useKVPair hook?

Reply:
This is a custom hook we'd expect a dev to write. This is also a 'moment in time', so we can discuss how to make writing this hook easier or less necessary